numl/NeutrinoML/HDF5Maker_module.cc

```diff
@@ -256,6 +274,45 @@ void HDF5Maker::analyze(art::Event const& e)
                              << "\nlocal plane " << wireid.Plane
                              << ", local wire " << wireid.Wire
                              << ", local time " << hit->PeakTime();
+                           
+
+
+    // Get the tpc geometry information from the id
+    if(std::find(tpc_ids_checked.begin(), tpc_ids_checked.end(), wireid.TPC ) != tpc_ids_checked.end())
+    {
+
+    }
```

Comment:
why the blank if block here?

Reply:
I added a comment here but the idea is that we only add the information to the table if we haven't done it already and in the case we have already checked it we just move on and do nothing.

Reply:
right, i think the logic itself makes sense. i'm more wondering why you choose to do
```
if(std::find(tpc_ids_checked.begin(), tpc_ids_checked.end(), wireid.TPC ) != tpc_ids_checked.end())
{
  // empty block
} else {
  [ code to add TPC to table ]
}
```
rather than simply
```
if (std::find(tpc_ids_checked.begin(), tpc_ids_checked.end(), wireid.TPC) == tpc_ids_checked.end())
{
  [ code to add TPC to table ]
}
```

Reply:
Ach 🤦 you're right for some reason my brain was so stuck in the first way I didn't even think to reverse the logic like that thank you!